docs/source-2.0/guides/model-translations/converting-to-openapi.rst

```diff
@@ -691,6 +691,29 @@ onErrorStatusConflict (``String``)
             }
         }
 
+.. _generate-openapi-setting-syncCorsPreflightIntegration:
+
+syncCorsPreflightIntegration (``boolean``)
+    Set to true to sync CORS preflight integration request templates with all combinations
+    of content-types from other methods within the same path resource.
+
+    .. code-block:: json
+        :caption: smithy-build.json
+
+        {
+            "version": "1.0",
+            "plugins": {
+                "openapi": {
+                    "service": "example.weather#Weather",
+                    "syncCorsPreflightIntegration": true
+                }
+            }
+        }
+
+    With this enabled, the ``passthroughBehavior`` for the CORS preflight integration
```

Comment:
Can there be a link to [the passThroughBehavior docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/integration-passthrough-behaviors.html) here?